### DISCUSSION TITLE
Thumnail not showing

### DISCUSSION THREAD

**Discussion by AleksandarNeustadt** [filamentphp/filament#6885] [General]
Hi, 
i try to make thumbnail with FIlament, and i use this code
Forms\Components\FileUpload::make('thumbnail'),

In form where i creating a post, i can see the thumnai (and upload work) but, when i save post, and i want to preview, i see only message 'loading...' and 'waiting for a size'

i see that thumnail is uploaded, and that is in database (name) but .... i cannot see the photo in prewiew 

Where is problem ?
Thank you

**Comment by pboivin** [comment]
Did you run `php artisan storage:link`?
https://laravel.com/docs/10.x/filesystem#the-public-disk

**Reply by rmitesh** [reply]
also, make sure that your `APP_URL` is valid
https://filamentphp.com/tricks/file-upload-previews-not-loading

**Comment by mujahidhasanwc** [comment]
it is valid as same as me
